repobee_sanitizer/_sanitize_repo.py

```diff
@@ -19,14 +19,15 @@
     _syntax,
     _format,
     _gitutils,
+    sanitizer,
```

Comment:
This introduces a circular import. `sanitizer` imports `_sanitize_repo`, and vice versa. That's in general not a good sign, and can sometimes cause the program to not be able to execute at all (infinite import loop).